configuration/configmgr/configmgrlib/mod_template_support/Operation.cpp

```diff
@@ -27,16 +27,16 @@ void Operation::addAttribute(modAttribute &newAttribute)
 }
 
 
-bool Operation::execute(EnvironmentMgr *pEnvMgr, Variables *pInputs)
+bool Operation::execute(EnvironmentMgr *pEnvMgr, Variables *pVariables)
 {
     bool rc = true;
     size_t count, startIndex;
 
-    pInputs->setInputIndex(0);
+    pVariables->setInputIndex(0);
 
     //
     // Determine the count of iterations to do
-    std::string countStr = pInputs->doValueSubstitution(m_count);
+    std::string countStr = pVariables->doValueSubstitution(m_count);
     try {
```

Comment:
Inconsistent style,  mixing  K&R with the preferred syntax

Reply:
Code changed to consistent pedantic representation.